### ISSUE TITLE
Set the number of digit/symbol chars

### ISSUE THREAD

**Issue by majkinetor** [mkropat/MlkPwgen#6]
Is this possible?

On semi related problem, how can I use predicate ?  I tried trivial :

```
New-PronounceablePassword -Length 20 -Predicate { $true }
```

but I got `Multiple ambiguous overloads found for "Generate" and the argument count: "3".`

**Comment by mkropat** [comment]
Thanks for the feedback. Right now there's no feature to set a required minimum or max number of characters in a given set.

> `Multiple ambiguous overloads found for "Generate" and the argument count: "3".`

Looks like this could be a bug. I'll try to take a look sometime this week.

**Comment by mkropat** [comment]
I fixed the `-Predicate` issue in [version 0.3.0](https://www.powershellgallery.com/packages/MlkPwgen/0.3.0). Give it a try and see if it works for you now.

It's not the most elegant, but you can use `-Predicate` to guarantee a string has a certain number of digits, for example:

```
PS > New-PronounceablePassword -Length 20 -Digits -Predicate { ($_ -replace '[^0-9]','').Length -eq 3 }
43RaInglivepily5Eder
```

**Comment by majkinetor** [comment]
This is fine. Thanks.
